presto-native-execution/scripts/setup-centos.sh

```diff
@@ -27,9 +27,6 @@ dnf install -y perl-XML-XPath
 
 python3 -m pip install regex pyyaml chevron black six
 
-# Required for Antlr4
```

Comment:
Is this no longer required? Would you mention this change in the commit message?

Reply:
We removed the Antlr dependency a while ago. We don't need this anymore. I will mention this in the commit message.